Cargo.toml

```diff
@@ -18,11 +18,15 @@ pin-project-lite = "0.2.12"
 
 [features]
 default = ["std"]
+loom = ["event-listener/loom"]
 std = ["event-listener/std"]
 
 [dev-dependencies]
 futures-lite = "2.0.0"
 
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(wasm_bindgen_unstable_test_coverage)'] }
```

Comment:
Huh, what's this about? Is that flag emitted by `wasm_bindgen_test::test`?

Reply:
Yeah I'm not sure about this one. @daxpedda What is the intended solution here?